go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/multiformats/go-multihash v0.2.3
 	github.com/onflow/atree v0.6.1-0.20230711151834-86040b30171f
-	github.com/onflow/cadence v1.0.0-M7
+	github.com/onflow/cadence v1.0.0-preview.2.0.20240216110422-a67dbbf21171
```

Comment:
This refers to a commit that isn't on any branch. 

It looks like this PR depends on https://github.com/onflow/cadence/pull/3111

Reply:
yeah, updated the description